libs/util/app-config/src/lib/app-config.ts

```diff
@@ -175,6 +179,9 @@ export function loadAppConfig() {
               SECONDARY_COLOR: parsedThemeSection.secondary_color,
               MAIN_COLOR: parsedThemeSection.main_color,
               BACKGROUND_COLOR: parsedThemeSection.background_color,
+              TITLE_COLOR: parsedThemeSection.title_color || 'black',
+              TITLE_HEADER_COLOR:
+                parsedThemeSection.title_header_color || 'black',
```

Comment:
For the new default theme I should probably set this to white by default.

Reply:
Yes indeed

Reply:
Also update the comment in the config please